src/main/java/ml/codeboy/thebot/commands/sound/AudioCommand.java

```diff
@@ -32,6 +32,11 @@ public void execute(CommandEvent event) {
 //        event.getManager().scheduler.setLatestEvent(event);//song info will now only be send if current is run
     }
 
+    /**
+     * Ensures that the user is in a voice channel
+     * @param event
+     * @return true if the bot is connected, false if the bot could not join
+     */
```

Comment:
Thanks for the documentation. Although maybe it should also mention that in case the bot isn't connected it also tries to connect